code/game/mecha/equipment/tools/medical_tools.dm

```diff
@@ -477,7 +477,7 @@
 	if(get_dist(src,A) >= 4)
 		occupant_message("The object is too far away.")
 		return 0
-	if(!A.reagents || ismob(A))
+	if(!A.reagents || ismob(A) || istype(A,/obj/item/reagent_containers/pill/floorpill)) //using an istype like this is bad, but adding a variable to every atom for one object is probably worse

```

Comment:
This could be a property/flag of reagents holder. Or omponent but that would need defining health/chem scan signals etc. (which is not the worst idea but more work)

Reply:
Didn’t think of that, I’ll go do that

Reply:
Didn’t think of that, I’ll make it a thing on the reagent holder

Reply:
The component is the better option honestly